lib/mayaUsd/fileio/orphanedNodesManager.cpp

```diff
@@ -41,7 +41,7 @@ namespace MAYAUSD_NS_DEF {
 // Class OrphanedNodesManager::Memento
 //------------------------------------------------------------------------------
 
-OrphanedNodesManager::Memento::Memento(Ufe::Trie<PullVariantInfo>&& pulledPrims)
+OrphanedNodesManager::Memento::Memento(PulledPrims&& pulledPrims)
```

Comment:
I added typedef and removed useless functions, so there is some noise in this file, sorry.